tests/ts_cluster_create_table_via_sql_SUITE.erl

```diff
@@ -0,0 +1,140 @@
+    C = client_pid(Ctx),
+    BadDDL =
+        io_lib:format(
+          "~s WITH (m_val = ~s)",
+          [ddl_common(), "plain_id"]),
+    Got = riakc_ts:query(C, BadDDL),
+    ?assertMatch({error, {1020, _}}, Got),
+    io:format("Not created table with bad properties: ~p\n", [Got]),
+    pass.
+
+create_test(Ctx) ->
+    C = client_pid(Ctx),
+    GoodDDL =
+        io_lib:format(
+          "~s WITH (n_val=~b, m_val = '~s')",
+          [ddl_common(), ?CUSTOM_NVAL, ?CUSTOM_MVAL]),
+    Got1 = riakc_ts:query(C, GoodDDL),
+    ?assertEqual({[],[]}, Got1),
+    io:format("Created table via query: ~p\n", [Got1]),
+    pass.
+
+re_create_fail_test(Ctx) ->
+    C = client_pid(Ctx),
+    Got = riakc_ts:query(C, ddl_common()),
+    ?assertMatch({error, {1014, _}}, Got),
+    io:format("Not created same table via query: ~p\n", [Got]),
+    pass.
+
+describe_test(Ctx) ->
+    C = client_pid(Ctx),
+    Qry = io_lib:format("DESCRIBE ~s", [ts_util:get_default_bucket()]),
+    Got = ts_util:single_query(C, Qry),
+    ?assertEqual(table_described(), Got),
+    pass.
+
+get_put_data_test(Ctx) ->
+    C = client_pid(Ctx),
+    Data = [[<<"a">>, <<"b">>, 10101010, <<"not bad">>, 42.24]],
+    Key = [<<"a">>, <<"b">>, 10101010],
+    ?assertMatch(ok, riakc_ts:put(C, ts_util:get_default_bucket(), Data)),
+    ?assertMatch({ok, {_, Data}}, riakc_ts:get(C, ts_util:get_default_bucket(), Key, [])),
+    io:format("Put a record, got it back\n", []),
+    pass.
+
+get_set_property_test(Ctx) ->
+    [Node1, Node2 | _] = ?config(cluster, Ctx),
+    ExpectedPL = unenquote_varchars(
+                   lists:usort(
+                     custom_bucket_properties())),
+    GetBucketPropsF =
+        fun(Node) ->
+                ActualProps =
+                    lists:usort(
+                      rpc:call(
+                        Node, riak_core_claimant, get_bucket_type,
+                        [list_to_binary(ts_util:get_default_bucket()), undefined, false])),
+                [PV || {P, _} = PV <- ActualProps, lists:keymember(P, 1, ExpectedPL)]
+        end,
+    ?assertEqual(ExpectedPL, GetBucketPropsF(Node1)),
+    ?assertEqual(ExpectedPL, GetBucketPropsF(Node2)),
+    io:format("Verified these properties are set: ~p\n", [ExpectedPL]),
+    pass.
+
+
+client_pid(Ctx) ->
+    client_pid(1, Ctx).
+client_pid(N, Ctx) ->
+    Nodes = ?config(cluster, Ctx),
+    rt:pbc(lists:nth(N, Nodes)).
+
+custom_bucket_properties() ->
+    [{n_val, ?CUSTOM_NVAL}, {m_val, ?CUSTOM_MVAL}].
+
+ddl_common() ->
+    ts_util:get_ddl(small).
+
+table_described() ->
+    {[<<"Column">>,<<"Type">>,<<"Is Null">>,<<"Primary Key">>, <<"Local Key">>],
+     [{<<"myfamily">>,   <<"varchar">>,   false,  1,  1},
+      {<<"myseries">>,   <<"varchar">>,   false,  2,  2},
+      {<<"time">>,       <<"timestamp">>, false,  3,  3},
+      {<<"weather">>,    <<"varchar">>,   false, [], []},
+      {<<"temperature">>,<<"double">>,    true,  [], []}]}.
+
+unenquote_varchars(PP) ->
```

Comment:
Of course it can, but there is an educational angle to it. This function is here to demonstrate the (non-intuitive) way single quotes are "escaped", and also to suggest that string values are parsed as varchars.